src/qmflows/packages/cp2k_mm.py

```diff
@@ -120,8 +121,12 @@ def run_job(cls, settings: Settings, mol: plams.Molecule,
 
         work_dir = work_dir if work_dir is not None else job.path
 
-        warnings = parse_output_warnings(job_name, r.job.path,
-                                         parse_cp2k_warnings, cp2k_warnings)
+        if validate_output:
```

Comment:
In practice this new keyword only affects `qmflows.cp2k` and `qmflows.cp2k_mm` (as of the moment at least).